proxmox/resource_vm_qemu.go

```diff
@@ -704,6 +710,7 @@ func resourceVmQemuRead(d *schema.ResourceData, meta interface{}) error {
 	d.Set("sshkeys", config.Sshkeys)
 	d.Set("ipconfig0", config.Ipconfig0)
 	d.Set("ipconfig1", config.Ipconfig1)
+	d.Set("ipconfig2", config.Ipconfig2)
```

Comment:
This needs the proxmox-api-go change merged in first... can you open a pull request for that?

Reply:
On it! Sorry, this is my first pull request ever.

Reply:
Done